pyaedt/edb_core/components.py

```diff
@@ -1009,18 +1017,18 @@ def disable_rlc_component(self, component_name):
         return False
 
     @pyaedt_function_handler()
-    def set_solder_ball(self, component, sball_diam="", sball_height=""):
+    def set_solder_ball(self, component="", sball_diam="100um", sball_height="150um"):
         """Set cylindrical solder balls on a given component.
 
         Parameters
         ----------
-        component : str or EDB component
+        componentname : str or EDB component
```

Comment:
@svandenb-dev If we modify this parameter name, we must change the parameter name in the method signature. Not sure if we need to add a warning for deprecation.
```suggestion
        component_name : str or EDB component
```